src/compas/geometry/transformations/rotation.py

```diff
@@ -69,25 +81,56 @@ def __init__(self, matrix=None, check=True):
                     raise ValueError('This is not a proper rotation matrix.')
         super(Rotation, self).__init__(matrix=matrix)
 
+    @property
+    def quaternion(self):
+        from compas.geometry import Quaternion
```

Comment:
same thing here.. why are these imports inlined?

Reply:
same

Reply:
this problem might go away after restructuring but that is for the next set of PRs :)

Reply:
👍